spec/services/smartview/oas3.fetched/spec_2.yaml

```diff
@@ -0,0 +1,543 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Smart View Environment APIs",
```

Comment:
This is an accidental duplicate of `spec_.yaml`. I copy/pasted from a browser view of each URL into these files which contained HTML forbidden messages.